Apps/GatewayApi/src/Controllers/UserProfileController.cs

```diff
@@ -389,6 +392,24 @@ public RequestResult<UserProfileModel> UpdateAcceptedTerms(string hdid, [FromBod
             return result;
         }
 
+        /// <summary>
+        /// Test US phone number validation required by the GatewayAPI and PHSA.
+        /// </summary>
+        /// <param name="phoneNumber">Phone number stripped of any masked characters.</param>
+        /// <returns>True if the submitted phone number conforms to US standards.</returns>
+        [HttpGet]
+        [Route("IsValidPhoneNumber/{phoneNumber}")]
+        [AllowAnonymous]
+        public RequestResult<bool> IsValidPhoneNumber(string phoneNumber)
```

Comment:
Can we change this to not use request result and not be anonymous?

Reply:
I'll look for the latest convention in the project and see make some changes. This will impact the front end as well but I'll get it done

Reply:
Returning ActionResult<bool> now.